test/cli/test_cli_flow.sh

```diff
@@ -60,9 +60,10 @@ function post_install_tests {
     bucketclass_cycle
     obc_cycle
     replication_cycle
-    check_backingstore
     check_deletes
     delete_replication_files
+    check_backingstore
```

Comment:
Why did you get it back here? 
The 2 last steps should be the deletes.

Please move it back up, and also the `check_dbdump`

Reply:
Done.
Not in the scope of this patch, but using a bash script as testing infrastructure is far from optimal and definitely not scaleable.
We need to create a different infrastructure IMO.